app/controllers/accept_account_invitations_controller.rb

```diff
@@ -0,0 +1,30 @@
+class AcceptAccountInvitationsController < ApplicationController
+  def show
+    not_found unless received_invitation &&
+                     invitation_belongs_to_current_user? &&
+                     not_accepted_invitation?
+  end
+
+  def update
+    received_invitation.update!(accepted_at: Time.now)
+    redirect_to account_path received_invitation.account_id
```

Comment:
use `()` - I'm having trouble reading it, is it `redirect_to account_path(received_invitation.account_id)` or `redirect_to(account_path received_invitation.account_id)`?